src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -447,6 +448,23 @@ async function validate_bucket_args(config_fs, data, action) {
     }
 }
 
+/**
+ * When setting notifications, we are supposed to send a test notification.
+ * If this test fails, we fail the user's request.
+ * @param {object} config_fs
+ * @param {object} notifications bucket's notification conf
+ * @returns Error encountered during notification test, if any
```

Comment:
Minor - I think it doesn't match the use of JSDoc `@returns` ([reference](https://jsdoc.app/tags-returns)).